scene/3d/area_3d.cpp

```diff
@@ -51,21 +51,30 @@ bool Area3D::is_gravity_a_point() const {
 	return gravity_is_point;
 }
 
-void Area3D::set_gravity_distance_scale(real_t p_scale) {
+void Area3D::set_gravity_point_distance_scale(real_t p_scale) {
 	gravity_distance_scale = p_scale;
 	PhysicsServer3D::get_singleton()->area_set_param(get_rid(), PhysicsServer3D::AREA_PARAM_GRAVITY_DISTANCE_SCALE, p_scale);
 }
 
-real_t Area3D::get_gravity_distance_scale() const {
+real_t Area3D::get_gravity_point_distance_scale() const {
 	return gravity_distance_scale;
 }
 
-void Area3D::set_gravity_vector(const Vector3 &p_vec) {
-	gravity_vec = p_vec;
-	PhysicsServer3D::get_singleton()->area_set_param(get_rid(), PhysicsServer3D::AREA_PARAM_GRAVITY_VECTOR, p_vec);
+void Area3D::set_gravity_point_center(const Vector3 &p_center) {
+	gravity_vec = p_center;
```

Comment:
As the value is now `gravity_point_center`, is not clearer to rename `gravity_vec`?

Reply:
I'm keeping `gravity_vec` because it's still the same member used for both point center and direction, and in the physics server both are linked to one single parameter `AREA_PARAM_GRAVITY_VECTOR`.